ddtrace/tracer/util_test.go

```diff
@@ -115,3 +116,35 @@ func TestParsePropagatableTraceTags(t *testing.T) {
 		})
 	}
 }
+
+func TestConvertHeaderToTag(t *testing.T) {
+
+	t.Run("standalone", func(t *testing.T) {
+		header, tag := ConvertHeaderToTag("header")
+		assert.Equal(t, header, "header")
+		assert.Equal(t, ext.HTTPRequestHeaders+".header", tag)
+	})
+
+	t.Run("mapped", func(t *testing.T) {
+		header, tag := ConvertHeaderToTag("header:tag")
+		assert.Equal(t, "header", header)
+		assert.Equal(t, "tag", tag)
+	})
+
+	// MTOFF - no info in spec about whitespaces between nonspace chars, e.g, "h e a d e r"
+	// or even before/after colon e.g, "header : tag"
```

Comment:
See: https://datadoghq.atlassian.net/wiki/spaces/APMINT/pages/2302444638/DD+TRACE+HEADER+TAGS#Tag-name
Unclear (to me) what the specifications are around casing.